debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -275,6 +275,16 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .withDefault(DEFAULT_SNAPSHOT_LOCK_TIMEOUT_MILLIS)
             .withDescription("The maximum number of millis to wait for table locks at the beginning of a snapshot. If locks cannot be acquired in this " +
                     "time frame, the snapshot will be aborted. Defaults to 10 seconds");
+    public static final Field INCLUDE_SCHEMA_CHANGES = Field.create("include.schema.changes")
+            .withDisplayName("Include database schema changes")
+            .withType(Type.BOOLEAN)
+            .withWidth(Width.SHORT)
+            .withImportance(Importance.MEDIUM)
+            .withDescription("Whether the connector should publish changes in the database schema to a Kafka topic with "
+                    + "the same name as the database server ID. Each schema change will be recorded using a key that "
+                    + "contains the database name and whose value includes the DDL statement(s)."
```

Comment:
DDL statement is not always correct, right?

Reply:
Fixed